.github/workflows/node-release.yml

```diff
@@ -49,7 +55,10 @@ jobs:
     if: ${{ needs.release-check.outputs.published == 'false' }}
     runs-on: ubuntu-latest
     permissions:
-      contents: write # for publishing release to GitHub
+      contents: write
```

Comment:
Let's keep these comments so folks in the future know why we need these permissions